src/MenuItems/MenuItems.js

```diff
@@ -0,0 +1,30 @@
+import MenuItem from "../MenuItem/MenuItem";
+
+import { useState } from "react";
+
+const MenuItems = ({menuItems}) => {
```

Comment:
`MenuItems` is a really good component name. I love this plural-singular naming pattern: `<MenuItems />` renders a list of components called `<MenuItem />`. I have used this a lot in my career and even used to make other people on my team use it as well :)